### ISSUE TITLE
[BUG]: Current documentation still refers to outdated 'path' in Middleware options instead of the new `pathPrefix`

### ISSUE THREAD

**Issue by aphexcx** [octokit/webhooks.js#946]
### What happened?

This does not work anymore:

```
import { createServer } from "node:http"
import { App, createNodeMiddleware } from "octokit"

const app = new App({
...
})

createServer(createNodeMiddleware(app.webhooks, { path: "/myapi/github/webhooks" })).listen(3001)
```
The middleware will reject all requests. 

This is because the options for Middleware have changed, [as per this documentation on octokit/app.js](https://github.com/octokit/app.js/?tab=readme-ov-file#createnodemiddlewareapp-options).  The above code must be changed to the following to work:
```
createServer(createNodeMiddleware(app, { pathPrefix: "/myapi/github" })).listen(3001)
```
Note that both the behavior (path is now a path prefix) *and* the default value (`/api/github/webhooks` -> `/api/github`) have changed!

This burned over five hours of my time! Please update the octokit/webhooks documentation and associated code & types.

### Versions

Octokit 3.1.2

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by wolfy1339** [comment]
The documentation for this repository is correct. The webhooks middleware expects a full path.

As for `@octokit/app` it expects a path prefix as described in its documentation.

**Comment by aphexcx** [comment]
I updated my comment - createNodeMiddleware(app) uses a pathPrefix, and  createNodeMiddleware(app.webhooks) from this library uses path. This is very confusing and should standardize on one or the either IMO.

**Comment by aphexcx** [comment]
Why are octokit/webhooks and octokit/app different?

**Comment by wolfy1339** [comment]
It shouldn't be as you aren't using the webhooks middleware at all if you are using the `@octokit/app` package.

It exports a middleware that incorporates the webhooks and oauth middlewares.

It's a separate package that has separate options.

This is how it always was implemented in this package, since 11.2.0
